tools/nl/translate_compare/translate_csvs.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the Licens
+
+"""Translate queries in csv files using the Google Translate API
+
+Translates a single column in CSV files specified by the user.
+Assumes a csv format where the 2 letter language codes are used
+as column headers in the first row. Will MODIFY input csv to add a
+column with the translations.
+
+Requires:
+    * A Google Cloud project with the Translate API enabled
+    * The environment variable PROJECT_ID=<gcloud project name>
+
+Example usage:
+    $ python3 translate_csvs.py path/to/dir/with/csvs \
+        --new_column_name="Chinese translation" \
+        --output_language="zh-CN" \
+        --source_column_index=2
+"""
+
+from absl import app, flags
+import glob
+import os
+import pandas as pd
+from google.cloud import translate
+
+CSV_FILE_EXTENSION = ".csv"
+DEFAULT_NEW_COLUMN_NAME = "translation"
+DEFAULT_OUTPUT_LANUGAGE_CODE = "en"
+
+project_id = os.environ.get("PROJECT_ID", "")
```

Comment:
Can we accept this as a flag instead of an env variable and default it to `datcom-204919`?

Reply:
Switched to a glag with default variable!